src/index.ts

```diff
@@ -56,6 +56,8 @@ export function makeMemlet(disklet: Disklet): Memlet {
    */
   const memletInstanceId = countOfMemletInstances++
 
+  let _nextFlushEvent: Promise<void> | undefined
```

Comment:
no reason to use the underscore convention since you don't even expose this param.